src/account/AccountUtils.ts

```diff
@@ -0,0 +1,216 @@
+  const proof = ZeroKnowledgeSig.deserialize(deserializer);
+  const verificationKeyHash = deserializer.deserializeOption("fixedBytes", 32);
+  return { jwt, uidKey, pepper, ephemeralKeyPair, proof, verificationKeyHash };
+}
+
+/**
+ * Utility functions for working with accounts.
+ */
+export namespace AccountUtils {
+  export function toBytes(account: Account): Uint8Array {
+    const serializer = new Serializer();
+    serializer.serializeU32AsUleb128(account.signingScheme);
+    account.accountAddress.serialize(serializer);
+    switch (account.signingScheme) {
+      case SigningScheme.Ed25519:
+        (account as Ed25519Account).privateKey.serialize(serializer);
+        return serializer.toUint8Array();
+      case SigningScheme.SingleKey: {
+        if (!isSingleKeySigner(account)) {
+          throw new Error("Account is not a SingleKeySigner");
+        }
+        const anyPublicKey = account.getAnyPublicKey();
+        serializer.serializeU32AsUleb128(anyPublicKey.variant);
+        switch (anyPublicKey.variant) {
+          case AnyPublicKeyVariant.Keyless: {
+            const keylessAccount = account as KeylessAccount;
+            serializeKeylessAccountCommon(keylessAccount, serializer);
+            return serializer.toUint8Array();
+          }
+          case AnyPublicKeyVariant.FederatedKeyless: {
+            const federatedKeylessAccount = account as FederatedKeylessAccount;
+            serializeKeylessAccountCommon(federatedKeylessAccount, serializer);
+            federatedKeylessAccount.publicKey.jwkAddress.serialize(serializer);
+            serializer.serializeBool(federatedKeylessAccount.audless);
+            return serializer.toUint8Array();
+          }
+          case AnyPublicKeyVariant.Secp256k1:
+          case AnyPublicKeyVariant.Ed25519: {
+            const singleKeyAccount = account as SingleKeyAccount;
+            singleKeyAccount.privateKey.serialize(serializer);
+            return serializer.toUint8Array();
+          }
+          default: {
+            throw new Error(`Invalid public key variant: ${anyPublicKey.variant}`);
+          }
+        }
+      }
+      case SigningScheme.MultiKey: {
+        const multiKeyAccount = account as MultiKeyAccount;
+        multiKeyAccount.publicKey.serialize(serializer);
+        serializer.serializeU32AsUleb128(multiKeyAccount.signers.length);
+        multiKeyAccount.signers.forEach((signer) => {
+          serializer.serializeFixedBytes(toBytes(signer));
+        });
+        return serializer.toUint8Array();
+      }
+      default:
+        throw new Error(`Deserialization of Account failed: invalid signingScheme value ${account.signingScheme}`);
+    }
+  }
+
+  export function toHexStringWithoutPrefix(account: Account): string {
+    return Hex.hexInputToStringWithoutPrefix(toBytes(account));
+  }
+
+  export function toHexString(account: Account): string {
+    return Hex.hexInputToString(toBytes(account));
+  }
```

Comment:
What do you think about just having `toHex`?
It would be consistent with `bcsToBytes()` and `bcsToHex()` we currently have.
And you can achieve both with `AccountUtils.toHex().toString()` or `AccountUtils.toHex().toStringWithoutPrefix()`